data/language/en-GB.txt

```diff
@@ -4188,6 +4188,16 @@ STR_5876    :{SMALLFONT}{BLACK}The engine to use for drawing the game.
 STR_5877    :Software
 STR_5878    :Software (hardware display)
 STR_5879    :OpenGL
+STR_5880    :Custom currency
+STR_5881    :Custom currency configuration
+STR_5882    :{WINDOW_COLOUR_2}Exchange rate: 
+STR_5883    :{WINDOW_COLOUR_2}is equivalent to {COMMA2DP32} GBP
```

Comment:
Explain GBP (Add the symbol)
E.g. 
{WINDOW_COLOUR_2}is equivalent to {COMMA2DP32} GBP (£)

As it's referenced as  ``Pounds (£)`` In-game